.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,29 +1,49 @@
 ## Problem
-*Describe the problem you're trying to solve, this should motivate why the changes you're proposing are needed.*
+*[Required] Describe the problem you're trying to solve, this should motivate why the changes you're proposing are needed.*
 
 ## Solution
-*Describe the solution you've created, elaborate on any technical choices you've made. Why is this the right solution and is a different solution not the right one?*
+*[Required] Describe the solution you've created, elaborate on any technical choices you've made. Why is this the right solution and is a different solution not the right one? What is the reasoning behind the chosen solution?*
 
 ## Issue tracker
-*Paste a link to the drupal.org issue queue item. If any other issue trackers were used, include links to those too.*
+*[Required] Paste a link to the drupal.org issue queue item. If any other issue trackers were used, include links to those too.*
+
+## Theme issue tracker
+*[Required if applicable] Paste a link to the drupal.org theme issue queue item. If any other issue trackers were used, include links to those too.*
 
 ## How to test
-*For example*
+*[Required] For example*
 - [ ] Using version X.Y.Z of Open Social with the example module enabled
 - [ ] As a sitemanager
 - [ ] Try to enable the option B on screen c/d/e
 - [ ] When saving I expect the result to be F but instead see G.
 - [ ] The expected result F is attained when repeating the steps with this fix applied.
 
+## Definition of done
+*[Required]*
+- [ ] Code/peer review is completed
+- [ ] All commit messages are clear and clean. If applicable a rebase was performed
+- [ ] All automated tests are green
+- [ ] Functional/manual tests of the acceptance criteria are approved
+- [ ] All acceptance criteria were meet
+- [ ] If applicable (I.E. new hook_updates) the update path from previous versions (major and minor versions) are tested
+- [ ] Code is tested on all branches that it has been cherry-picked
```

Comment:
We might need to split this up before / after merge. 
Testing this on all cherry-picked branches can only be done when cherry-picked and only if applicable. 
Update hook numbers also might need to be adjusted on cherry-pick so we can also include that.